### ISSUE TITLE
Feature requests for API browser docs site re giving Feedback and Constructor/Method signatures

### ISSUE THREAD

**Issue by ericsampson** [dotnet/dotnet-api-docs#7835]
1) It would be excellent if each page had a `Feedback` link that would create a prepopulated GitHub issue, just like the main Microsoft docs site pages do.

2) For the Constructor and Methods signatures shown in the various pages, would it be possible to include the parameter names instead of just the data types? Often, just seeing the data types isn't enough to get a sense of what's going on, because you see parameter lists like `(string, Someclass, string, string, bool, string)` are not very useful to the person browsing the site :)

Thanks!

cc @IEvangelist

**Comment by IEvangelist** [comment]
Hi @ericsampson,

Thank you so much for posting this issue. We really appreciate it!

I discovered that the feedback feature is intentionally turned off for API references, I'm not certain why. I can tell you that I do know that there is an issue that @gewarren created #7451 that aims to address this general issue. It's something that we've also heard from other customers, and even educators. We're working to add better documentation for the top 10% of traffic for APIs. If there are any specific examples that you'd like to see, we could cross-reference it to see if it is part of this effort, if it is not maybe we could create an individual PR for your specific use case. Sound reasonable?

**Comment by gewarren** [comment]
Thanks for the feedback @ericsampson. 

The feedback mechanism was turned off because we simply don't have the resources to respond to every issue. But you are free to create issues, like you did.

Regarding the method signatures, I think you're referring to the Overloads list and also the individual headings, because the syntax blocks themselves do show the parameter names. Can you confirm?

![image](https://user-images.githubusercontent.com/24882762/158857543-e1cd2c4b-4e50-4859-bf59-4b4f67ddaa10.png)

Example page: https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.partitioner.create?view=net-6.0

**Comment by ericsampson** [comment]
Hi Genevieve, thanks! (great name, it's one of the names in my family  :)

You have it exactly right. For both `Constructors` and `Methods`, it's the Overload list, the individual headings (and so the TOC on the right).

Issue #7451 seems great, having cross-linking between the API browser content and the conceptual docs would be HUGELY USEFUL :)

Thanks and cheers!

**Comment by gewarren** [comment]
@ericsampson I can see how the parameter names could be useful in the Overloads list. However, adding them to the headings would make the TOC on the right harder to read, IMO, because the headings would start to wrap more, and it would be harder to count the number of parameters at a glance. Unfortunately, I think the overloads list is just displaying the headings, so they are interlinked, but perhaps we can change the link text in the Overloads list. Thoughts @mimisasouvanh @huangmin-ms?

**Comment by ericsampson** [comment]
I agree that keeping the TOC on the right in its present (short) form would be a good idea.
